### ISSUE TITLE
[lhCpp] Do not expand "else" when it precedes an "if"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/lh-vim#7]
```
===What steps will reproduce the problem?===
1. type "if foo<esc>"
2. continue with "Ielse<space>"

===What is the expected output? ===
{{{
        else if (foo)
        {
            «»
        }
        «»
}}}

===What do you see instead?===
{{{
        else
        {

        }
        «»if (foo)
        {
            «»
        }
        «»
}}}
```

Original issue reported on code.google.com by `luc.herm...@gmail.com` on 1 Oct 2008 at 2:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `luc.herm...@gmail.com` on 23 Oct 2008 at 3:24
- Added labels: **lhCpp**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision r#332
```

Original comment by `luc.herm...@gmail.com` on 6 Jan 2011 at 2:15
- Changed state: **Fixed**
